models/articles.models.js

```diff
@@ -18,7 +18,27 @@ exports.setArticles = () => {
     `;
 
   return db.query(formatedArticles).then(({ rows }) => {
-    console.log(rows);
     return rows;
   });
 };
+
+exports.selectArticleById = (article_id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1 ", [article_id])
+    .then(({ rows }) => {
+      return rows[0];
+    });
```

Comment:
According to the pr for task 4 - it's better to return the Promise.reject in the model instead of dealing with it in the controller. 
Also if dealing with it here, then for getting all the comments, selecArticleById can be invoked in controller.